### ISSUE TITLE
DRA not defined error

### ISSUE THREAD

**Issue by shwetankverma28** [BradyPlanden/LiiBRA.jl#29]
I am trying to run the code given in file LiiBRA_EX.jl in the examples folder but I keep getting the following error:
Code where I get the error:
```
function Realise(Cell, SList)
    A = B = C = D = tuple()
    for i in SList 
        #Arrhenius
        Cell.Const.T = 298.15
        Arr_Factor = (1/Cell.Const.T_ref-1/Cell.Const.T)/R

        #Set Cell Constants
        Cell.Const.SOC = i
        Cell.Const.κ = Cell.Const.κf(Cell.Const.ce0)*exp(Cell.Const.Ea_κ*Arr_Factor)
        Cell.RA.Nfft = Cell.RA.Nfft!(Cell.RA.Fs, Cell.RA.Tlen)
        Cell.RA.f = Cell.RA.f!(Cell.RA.Nfft)
        Cell.RA.s = Cell.RA.s!(Cell.RA.Fs,Cell.RA.Nfft,Cell.RA.f)

        #Realisation
        Aϕ, Bϕ, Cϕ, Dϕ = DRA(Cell,Cell.RA.s,Cell.RA.f)

        #Flatten output into Tuples
        A = flatten_(A,Aϕ)
        B = flatten_(B,Bϕ)
        C = flatten_(C,Cϕ)
        D = flatten_(D,Dϕ)
    end
return A, B, C, D
end
```
Error:

```
ERROR: UndefVarError: DRA not defined
Stacktrace:
 [1] Realise(Cell::LiiBRA.Params, SList::Vector{Float64})
   @ Main .\Untitled-1:24
 [2] top-level scope
   @ Untitled-1:50

Realise (generic function with 1 method)
```

I am using Julia 1.7.0

**Comment by BradyPlanden** [comment]
Hi @shwetankverma28, 

Good catch, yes, this example is out of date. The original DRA() function has been merged with the CI-DRA function in breaking release 0.2.0. Updating this example should be included in release 0.3.2.

For the time being, the example in the readme captures the same functionality as this example.

**Comment by shwetankverma28** [comment]
Hi @BradyPlanden ,
Thanks for your reply. Can you also add examples for running cycling tests as well as ways to add degradation modes to the mode.
